sdcm/send_email.py

```diff
@@ -63,7 +63,8 @@ def prepare_email(self, subject, content, recipients, html=True, files=()):  # p
         msg = MIMEMultipart()
         msg['subject'] = subject
         msg['from'] = self.sender
-        msg['to'] = ','.join(recipients)
+        if recipients:
```

Comment:
why not checking all  msg['subject'], msg['from']??
I think we shouldn't check it here. We shouldn't run the method when recipients is empty

Reply:
it is done to be able to test whole report cycle except actual email sending, by setting recipient to None.